pkg/tools/log/spanlogger/spanlogger.go

```diff
@@ -36,11 +36,44 @@ const (
 	dotCount        int = 3
 )
 
+type logLevel int
+
 // spanlogger - provides a way to log via opentracing spans
 type spanLogger struct {
 	span    opentracing.Span
 	entries map[interface{}]interface{}
 	lock    sync.RWMutex
+	level   logLevel
```

Comment:
@Mixaster995 , @glazychev-art Do we really need logLevel for spanLogger?

Reply:
Span doesn't have logLevel. So I think it can be useful to limit the output at different levels...